src/App.vue

```diff
@@ -36,12 +36,14 @@
 			</NcCheckboxRadioSwitch>
 
 			<span class="form">
-				<NcMultiselect v-model="country"
+				<NcSelect v-model="country"
+					:multiple="true"
```

Comment:
I'm pretty sure this should not be a multiselect even if it was called so before.. You can select one country.
```suggestion
				<NcSelect v-model="country"
```